src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/events/CcdUpdater.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.events;
+
+import com.google.common.base.Strings;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.CaseRetriever;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.model.Envelope;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+@Service
+public class CcdUpdater {
```

Comment:
This used to be `EventPublisherContainer`. Code structure is pretty much the same, but now, instead of returning functions that update CCD, it updates CCD itself.